src/SFA.DAS.EmployerIncentives.Domain.UnitTests/IncentiveApplicationTests/WhenAnApprenticeshipIsCreated.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using AutoFixture;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.Common.Domain.Types;
+using SFA.DAS.EmployerIncentives.Domain.Factories;
+
+namespace SFA.DAS.EmployerIncentives.Domain.UnitTests.IncentiveApplicationTests
+{
+    public class WhenAnApprenticeshipIsCreated
+    {
+        private Fixture _fixture;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _fixture = new Fixture();
+        }
+
+        [Test]
+        public void Then_the_total_incentive_amount_is_the_24_or_under_total_if_the_apprentice_is_exactly_24_at_the_start_date()
```

Comment:
Test name is slightly ambiguous here - the test date of birth is on the boundary of one day less than 25 years old